lib/llvm-backend/src/code.rs

```
@@ -6853,7 +6862,9 @@ impl FunctionCodeGenerator<CodegenError> for LLVMFunctionCodeGenerator {
                 state.push1(result.try_as_basic_value().left().unwrap());
             }
             _ => {
-                unimplemented!("{:?}", op);
```

`unreachable!("Operator {:?} not handled", op);`

---

This one should return an Error i think since we are not supporting all new opcode in future proposal.

---

OK.